backend/src/utils/agent.py

```
@@ -8,12 +8,13 @@
 from langchain_arcade import ArcadeToolManager
 from psycopg.connection_async import AsyncConnection
 from sqlalchemy import text
+from langgraph.errors import NodeInterrupt
 
 from src.repos.thread_repo import ThreadRepo
 from src.entities.a2a import A2AServer
 from src.services.mcp import McpService
 from src.repos.user_repo import UserRepo
-from src.constants import APP_LOG_LEVEL, UserTokenKey
+from src.constants import APP_LOG_LEVEL, ARCADE_API_KEY, UserTokenKey
```

Ruff (0.11.9)

`src.constants.UserTokenKey` imported but unused

Remove unused import: `src.constants.UserTokenKey`

(F401)